corehq/util/test_utils.py

```diff
@@ -232,6 +234,23 @@ def inner(*args, **kwargs):
     return inner
 
 
+class OverridableSettingsTestMixin(object):
```

Comment:
@dimagi/team-commcare-hq FYI if you want to run tests that use `setUpClass` / `tearDownClass` but also want to override settings.